components/lib/dataprotect/src/main/java/mozilla/components/lib/dataprotect/SecureAbove22Preferences.kt

```diff
@@ -181,6 +186,7 @@ private class SecurePreferencesImpl23(context: Context) : KeyValuePreferences {
      * which stores it in system's secure storage layer exposed via [AndroidKeyStore].
      */
     private fun generateManagedKeyIfNecessary() {
+        // Do we need to check this on every access, or just during instantiation? Is the overhead here worth it?
```

Comment:
We could probably just do this during init because it seems unlikely/impossible that you could lose this key mid app run ?

Reply:
That's what I'm hoping for. Let's address this in a follow-up.